### DISCUSSION TITLE
Synchronize data to QFieldCloud causes program to sign out when switching between devices

### DISCUSSION THREAD

**Discussion by AnomalousExploration** [opengisch/QField#3943] [Q&A]
Hi,

I've been experimenting with your beta version and am hoping to use QFieldCloud for our field season. I have some questions regarding data collection.

I would like to have multiple people collecting data in the same project during the day. At the end of the day, I would like to upload their information to the cloud and be able to sync it back with updated information for the following day. I went with the pro plan to do this. I want to oversee all data from the cloud and the desktop as the main user. I'm newer to your terminology and their may be a better set up and I may be setting up my system inappropriately. Ideas are welcomed! Questions:

1. If multiple devices upload unique file versions to the cloud, will it be able to continuously update the project file automatically incorporating in all of their data? Currently I'm worried this process will erase previous versions. Is there a way to do this smoothly?

2. If multiple users are on the same account in qfield, when you 'push changes' to cloud it seems that you need to re-sign in with each device. Is it possible to fix this?

3. It seems as though everytime I add data from the desktop or from a data phone, I make a new version of the project...potentially with a whole new set of all files. This quickly quadruples the number of file versions stored in the cloud. Is there an efficient way to keep only the current version of the file or to quickly delete all older versions?

Thank-you for your time. I really appreciate the software and your work!
Ellen

**Comment by ZsanettMed** [comment]
Hi Ellen,

Short answer to your mail: What you would like to do is exactly what QFieldCloud is made for, but for that, you have to create an organization with multiple users. 
Long version: I would recommend you to read the documentation (get started, config your projects, sync projects, collect data, push changes and so on): https://docs.qfield.org/get-started/tutorials/get-started-qfc/
and
https://qfield.cloud/faq.html

I am sure you will find all the information you are looking for. If not, feel free to ask further questions! :) 
Best, 
Zsanett
